coderd/appearance/appearance.go

```diff
@@ -10,12 +10,7 @@ type Fetcher interface {
 	Fetch(ctx context.Context) (codersdk.AppearanceConfig, error)
 }
 
-var DefaultSupportLinks = []codersdk.LinkConfig{
-	{
-		Name:   "Documentation",
-		Target: "https://coder.com/docs/coder-oss",
-		Icon:   "docs",
-	},
+var defaultSupportLinks = []codersdk.LinkConfig{
```

Comment:
This probably doesn't need to be a var since it's dynamic below anyways

Reply:
This is updated!